mteb/abstasks/MTEBResults.py

```diff
@@ -19,6 +19,7 @@
 
 Split = str
 ScoresDict = Dict[str, Any]
```

Comment:
Would it not make sense to have at least some validation code for this?
I also think it would be nice to add a `version` field to all scores that we export (not just MTEB version but format version).
That way we will know how to parse it and convert between the different score dicts we might come up with in the future.

Reply:
I believe mteb version == format version. We might of course load in old format files and rewrite them to the new format. In which case I would just attempt loading it with the current expected format and if it fails attempt to use the mteb version.